DecipherNow/gm-fabric-dashboard
# Fix: Table interactions

Resolves #1180 

## Proposed Changes

  - Edited props in TableRow.js to make more semantic sense. `isOpen` now refers to the state of the drawer and `selectable` refers to whether the row has something for the user to interact with.
  - Moved ThreadsTable from index.js to ThreadsTable.js
 
  
## How to Test

  - Navigate to JVM => Threads. Confirm that rows without drawers (no stack trace) do not have a hover state and that the cursor is default. You can also double check other tables around the app if you feel so inclined.
